ceph_exporter/ceph/commands/ceph_df.py

```diff
@@ -28,6 +28,7 @@
 from ..metrics import ceph_pool
 from ..metrics import ceph_storage_bytes
 
+
```

Comment:
I'm not against using PyLint or something else to clean up whitespace etc., but that should be handled as a separate commit so that more substantive changes don't get overlooked.